stub-environment/roles/BCPC-Hadoop-Head-HBase.json

```diff
@@ -3,7 +3,9 @@
   "json_class": "Chef::Role",
   "run_list": [
     "role[Basic]",
+    "recipe[bcpc-hadoop::phoenixqs_hbase_site]",
     "recipe[bcpc-hadoop::hbase_master]",
+    "recipe[bcpc-hadoop::phoenix_query_server]",
     "recipe[bcpc-hadoop::hbase_repl]",
     "recipe[bcpc::diamond]"
   ],
```

Comment:
We should have PQS behind ``haproxy``` or something similar for service availability since any node can go into maintenance.

Reply:
PQS uses zookeeper for location and load balancing, we can add haproxy in a follow up I suppose